### DISCUSSION TITLE
Get the correct status from %post Area.

### DISCUSSION THREAD

**Discussion by tracyluffy24** [rpm-software-management/rpm#2491] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/131222833/233004934-c82ddd32-328a-4d20-8ded-5e9a194d14be.png)

Run exit 1 in the %post part of the package.spec file. The result is "warning: %post(xxx) scriptlet failed, exit status 1" . 
The command output obtained by using shell command 
![image](https://user-images.githubusercontent.com/131222833/233005152-94a9085f-465d-4823-bad5-c27689f8e4eb.png)
 is 0.  How can I use 
![image](https://user-images.githubusercontent.com/131222833/233005179-aeec4cf2-dc13-4579-9bbe-1709487ff4d5.png)
 to obtain the correct exit result: 1.

**Comment by pmatilai** [comment, marked as answer]
It's not possible in current versions. Only pre(un)-status is considered, and failure for those actually prevents the install/erase from taking place at all. 

What seems like an obvious thing with an isolated package, a single return code becomes far less meaningful in a large transaction, often with multiple not strictly related packages.

**Reply by tracyluffy24** [reply]
Does the new version，RPM V6 support this function?  Looking forward to your reply， thanks very much!
